tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -531,3 +550,40 @@ func deleteStoreFromSchedulerConfig(cmd *cobra.Command, schedulerName string, ar
 	}
 	cmd.Println("Success!")
 }
+
+func showShuffleRegionSchedulerRolesCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 0 {
+		cmd.Println(cmd.UsageString())
+		return
+	}
+	path := path.Join(schedulerConfigPrefix, cmd.Parent().Name(), "roles")
+	r, err := doRequest(cmd, path, http.MethodGet)
+	if err != nil {
+		cmd.Println(err)
+		return
+	}
+	cmd.Println(r)
+}
+
+func setSuffleRegionSchedulerRolesCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		cmd.Println(cmd.UsageString())
+		return
+	}
+	var roles []string
+	fields := strings.Split(strings.ToLower(args[0]), ",")
+	for _, f := range fields {
+		if f != "" {
+			roles = append(roles, f)
+		}
+	}
+	b, _ := json.Marshal(roles)
```

Comment:
Do we need to handle this error?

Reply:
it should not return any error as it only contains basic types.